### ISSUE TITLE
Support exporting metrics to Stackdriver Logging

### ISSUE THREAD

**Issue by johnsonj** [cloudfoundry-community/stackdriver-tools#99]
Today we [split](https://github.com/cloudfoundry-community/stackdriver-tools/blob/master/src/stackdriver-nozzle/nozzle/nozzle.go#L79) events from the Loggregator into Stackdriver Logging/Monitoring. It is desirable to have the metric data available in Stackdriver Logging so users can use exports for further analysis. An example would be exporting to BigQuery to perform custom calculations on metrics not available in Stackdriver Monitoring. 

Design Considerations:
- Should it be a 'All or Nothing' to log to both Stackdriver Logging and Monitoring or should each endpoint have it's own list of events? 
- Should we need to perform the same culling for metrics destined for Stackdriver Logging?

**Comment by mrdavidlaing** [comment]
> Should it be a 'All or Nothing' to log to both Stackdriver Logging and Monitoring or should each endpoint have it's own list of events?

We were certainly surprised that when adding `CounterEvent,ValueMetric,ContainerMetric` events to [`firehose.events:`](https://github.com/cloudfoundry-community/stackdriver-tools/blob/master/jobs/stackdriver-nozzle/spec#L21) sent them to a different place ( Stackdriver Monitoring rather than Stackdriver Logging ).

We'd like to propose having 2 separate config settings ( in both the BOSH release and the Tile )

```
  firehose.events_to_stackdriver_logging:
    description: Whitelisted event types to send to Stackdriver Logging (comma separated). Valid values include LogMessage,Error,HttpStartStop,CounterEvent,ValueMetric,ContainerMetric
    default: LogMessage,Error
  firehose.events_to_stackdriver_monitoring:
    description: Whitelisted event types to send to Stackdriver Monitoring (comma separated). Valid values include CounterEvent,ValueMetric,ContainerMetric
    default: CounterEvent,ValueMetric,ContainerMetric
```

For the Pivotal Tracker on GCP use case; we'd probably send everything to Stackdriver Logging -> because we want the ability to send some of the data into BigQuery, and `CounterEvent,ValueMetric` to Stackdriver Monitoring

**Comment by mrdavidlaing** [comment]
@johnsonj Thanks for acting on this issue so quickly.

Will this change make its way into v1.0.6 of the [GCP Stackdriver Nozzle for PCF tile](https://network.pivotal.io/products/gcp-stackdriver-nozzle) ?

**Comment by johnsonj** [comment]
Yes! I just need to do some manual validation before releasing. I think it
will be v1.1.0.

**Comment by johnsonj** [comment]
Released as [v1.0.6](https://github.com/cloudfoundry-community/stackdriver-tools/releases/tag/v1.0.6). Pivnet update pending approval.

**Comment by johnsonj** [comment]
Re-opening since v1.0.6 was rolled back due to performance issues

**Comment by johnsonj** [comment]
Released as [v2.0.1](https://github.com/cloudfoundry-community/stackdriver-tools/releases/tag/v2.0.1).
